### PR TITLE
MediaConch license change

### PR DESCRIPTION
**Pull request by JeromeMartinez**

### CODE REVIEW

File: src/BlogBundle/Posts/2018-03-19-Why-we-changed-MediaConch-license.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
... with a permissive license is that we don’t like to constrain anyone: we want to convince everyone that open source is the way to go. From our point of view, you should choose open source because you are convinced that this is the best method, not because you are forced to due to the license. Ha, another point: We also want you choose us because you think we are the best for doing the job :), not because we block others. For PREFORMA, we were the only ones able to relicense MediaInfo for complying with PREFORMA constraints, and this is something we were not comfortable with. We changed the license to the one we usually use and we explicitly authorize you to reuse our code for projects requiring a specific license that would permit anyone to have the same right as us when applying to a Call for Proposals requiring a specific license. MediaConch (as well as MediaInfo) are now available as your choice: under a BSD 2-Clause license (the “default” license), and/or Apache License, and/or GNU Lesser General Public License 2.1 or later (which can be transformed to nearly any modern GNU license), and/or GNU General Public License 2.0 or later, and/or Mozilla Public License 2.0 and/or later.
```
Comment (ablwr) [inline src/BlogBundle/Posts/2018-03-19-Why-we-changed-MediaConch-license.md]: I would drop the "Ha, another point:" but up to you

**Comment by ablwr** [comment]
I hate seeing the "smart quotes" especially when there are some "dumb quotes" mixed in so I prefer a find-replace to make them all "dumb quotes." Made a couple of comments (as single-comments and not review, sorry) but mostly LGTM.

**Comment by JeromeMartinez** [comment]
All remarks handled.

**Review by ablwr — APPROVED** [review]

File: src/BlogBundle/Posts/2018-03-19-Why-we-changed-MediaConch-license.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
... library we would use in MediaConch as a dependency. If you know about open source licensing, you can guess the origin of a (not-so-small) problem… it would have meant that nearly no library would have been usable, so we would have had to develop everything, including some features like a database, an XML parser, et cetera, completely "from scratch"… when these features are already covered by well known open source libraries. Lot of open source libraries exist but would not be usable because the license was “wrong". We would have pulled ourselves out of the open source ecosystem -- if we were to say to open source advocates that we can’t use a library because it is BSD/MIT/Apache licensed, we would get strange looks from our peers in open source communities. At this early design and proposal phase during the PREFORMA project, we were 6 organizations in competition with each other, to win the bid to develop the file format validation software. All six competitors agreed that we would quit the project if this unnecessary license restriction was not lifted. The response was that we could use third party open source libraries for “non-core" features, and this crisis was averted (phew).
```
Comment (dericed) [inline src/BlogBundle/Posts/2018-03-19-Why-we-changed-MediaConch-license.md]: Given our mixed audience I worry that the library in "nearly no library would have been usable" could be misinterpreted. Maybe 'software library' at least in the first usage.

Comment (dericed) [reply src/BlogBundle/Posts/2018-03-19-Why-we-changed-MediaConch-license.md]: "All six competitors agreed that we would quit the project if this unnecessary license restriction was not lifted." Was this shared, I remember expressing a shared concern but not certain if all competitors agreed to quitting under those terms.

File: src/BlogBundle/Posts/2018-03-19-Why-we-changed-MediaConch-license.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+We did not understand why this license change unblocked things, as we still own the copyright so it is obvious that this constraint was only temporary.
+
+The PREFORMA project is now finished. The agreement we had is over (since December 31, 2017). There is no long term agreement about the license of MediaConch and we own the copyright. We don’t want that the tool is not spread due to this weird choice of license, We don't want the tool to not spread due to this weird choice of license and we don't want someone to be blocked due to a problem with the license, so we will release MediaConch with the license we usually use, that’s all..
```
Comment (dericed) [inline src/BlogBundle/Posts/2018-03-19-Why-we-changed-MediaConch-license.md]: double negative "We don’t want that the tool is not spread"

Comment (dericed) [reply src/BlogBundle/Posts/2018-03-19-Why-we-changed-MediaConch-license.md]: also this paragraph includes a repeated phase

File: src/BlogBundle/Posts/2018-03-19-Why-we-changed-MediaConch-license.md (hunk 2 above)
Comment (JeromeMartinez) [reply src/BlogBundle/Posts/2018-03-19-Why-we-changed-MediaConch-license.md]: It was implicit if I remember well. OK for rewording it and not doing it explicit.

**Pull request merged**